crates/header-translator/src/config.rs

```diff
@@ -582,6 +606,9 @@ pub struct MethodData {
     #[serde(rename = "unsafe")]
     #[serde(default)]
     pub unsafe_: Unsafe,
+    #[serde(rename = "override")]
+    #[serde(default)]
+    pub override_: Option<MethodOverride>,
```

Comment:
Nit: I'd avoid this struct altogether, everything in here is an "override". Just inline `pub return_: Option<ReturnOverride>,` here (or maybe `pub return_: ReturnOverride`, since it impls Default).